package.json

```diff
@@ -64,10 +64,11 @@
     "jest": "^26.6.3",
     "lint-staged": "^11.0.0",
     "npm-run-all": "^4.1.5",
-    "prettier": "^2.3.0",
-    "webpack": "^4.46.0"
+    "nyc": "^15.1.0",
+    "prettier": "^1.19.1",
```

Comment:
Why downgrading `prettier`?

Reply:
oh yep it shouldn't, somehow other things got changed too 😞 will fix